src/main/java/org/spongepowered/mod/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -225,4 +234,41 @@ public void shutdown(Text kickMessage) {
 
         initiateShutdown();
     }
+
+    private final Map<String, ChannelListener> channelListeners = new HashMap<String, ChannelListener>();
+
+    @Override
+    public void registerChannel(Object plugin, ChannelListener listener, String channel) throws ChannelRegistrationException {
+        checkNotNull(plugin, "plugin");
+        checkNotNull(listener, "listener");
+        checkNotNull(channel, "channel");
+        checkArgument(SpongeMod.instance.getGame().getPluginManager().fromInstance(plugin).isPresent(),
+                "Provided plugin argument is not a plugin instance");
+        if (IMixinServer.vanillaChannels.contains(channel)) {
```

Comment:
Thinking about it, this doesn't make sense either, because it would prevent a plugin from listening to a vanilla channel.